osutil/user_test.go

```diff
@@ -137,6 +178,9 @@ func (s *createUserSuite) TestAddUserInvalidUsername(c *check.C) {
 }
 
 func (s *createUserSuite) TestAddUserWithPassword(c *check.C) {
+	r := osutil.MockhasAddUserExecutable(func() bool { return true })
```

Comment:
There are other test that looks as if they ought to have osutil.MockhasAddUserExecutable?

Reply:
I checked the tests again and added it to few more places.